decidim-core/config/locales/en.yml

```diff
@@ -947,6 +947,13 @@ en:
       notification_event:
         notification_title: An event occurred to <a href="%{resource_path}">%{resource_title}</a>.
       reports:
+        parent_hidden:
+          email_intro: A %{resource_type} you have created was hidden because its parent resource was also hidden.
```

Comment:
Check out this comment from @carolromero 

https://github.com/decidim/decidim/issues/14448#issuecomment-2768747699

We're actively trying to not talk to participants about "parent resources" because that's Klingon for them. Can you change the strings please? Thanks

Reply:
I have added d92b5d51e4, that fixes the notification.
![image](https://github.com/user-attachments/assets/978bf80a-44bd-4362-90c4-1a8caf2e95aa)